java/src/com/ibm/streamsx/topology/TStream.java

```diff
@@ -1051,6 +1051,16 @@
      */
     TStream<T> parallel(Supplier<Integer> width, Function<T,?> keyer);
     
+    /**
+     * Sets the current stream as the start of a parallel region.
```

Comment:
Doesn't it set the logic that produced this stream as the start of a parallel region?

Reply:
"The logic that produced this stream" isn't stream-centric wording. What about 

```
"includes this stream as the start of a parallel region."
```